plugins/woocommerce-admin/client/inbox-panel/test/index.js

```diff
@@ -121,3 +156,54 @@ describe( 'inbox_panel_view_event', () => {
 		} );
 	} );
 } );
+
+describe( 'inbox_note_view event', () => {
+	test( 'should fire when inbox note card calls onNoteVisible', () => {
+		notes.forEach( ( note ) => ( note.is_deleted = false ) );
```

Comment:
Can we change the test to work with a copy of the `notes` array? It really shouldn't have a side effect like this.